src/main/java/uk/gov/cshr/civilservant/controller/CivilServantController.java

```diff
@@ -29,17 +38,29 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CivilServantController.class);
 
+    private LineManagerService lineManagerService;
+
+
+    private IdentityService identityService;
+
     private CivilServantRepository civilServantRepository;
 
+    private NotifyService notifyService;
+
     private RepositoryEntityLinks repositoryEntityLinks;
 
+
     @Autowired
     public CivilServantController(CivilServantRepository civilServantRepository,
-                                  RepositoryEntityLinks repositoryEntityLinks) {
+                                  RepositoryEntityLinks repositoryEntityLinks,
+                                  LineManagerService lineManagerService) {
         checkArgument(civilServantRepository != null);
         checkArgument(repositoryEntityLinks != null);
         this.civilServantRepository = civilServantRepository;
         this.repositoryEntityLinks = repositoryEntityLinks;
+
```

Comment:
STYLE - Why several empty lines all of a sudden?

Reply:
Probably happened in rebase. removed